python/ray/experimental/sgd/pytorch/pytorch_runner.py

```diff
@@ -177,20 +224,44 @@ def get_state(self):
             for k, v in state_dict.items():
```

Comment:
This whole loop body could be a dict comprehension:
```
cpu_state_dicts.append({k: v.cpu() for k, v in model.state_dict().items()})
```

Reply:
adjusted